kse/src/main/resources/org/kse/gui/dialogs/sign/resources.properties

```diff
@@ -44,7 +44,7 @@ DSignCrl.Title                         = Sign CRL
 DSignCrl.ValReqCrlNumber.message       = A value is required for CRL Number
 DSignCrl.jbCancel.text                 = Cancel
 DSignCrl.jbOK.text                     = OK
-DSignCrl.jcbSignatureAlgorithm.tooltip = Signature algorithm to sign CRL with
+DSignCrl.jcbSignatureAlgorithm.tooltip = Signature algorithm used to sign the CRL
```

Comment:
The old version was not wrong, but maybe the new version is better... I can't tell...